pkg/testcase/secretsencrypt.go

```diff
@@ -22,84 +24,91 @@ func TestSecretsEncryption() {
 	errSecret := shared.CreateSecret("secret1", "default")
 	Expect(errSecret).NotTo(HaveOccurred(), "error creating secret")
 
-	shared.LogLevel("info", "TEST: 'CLASSIC' Secrets Encryption method")
-
 	index := len(nodes) - 1
 	cpIp := nodes[index].ExternalIP
-	secretsEncryptOps("prepare", product, cpIp, nodes)
-	secretsEncryptOps("rotate", product, cpIp, nodes)
-	secretsEncryptOps("reencrypt", product, cpIp, nodes)
 
-	if strings.Contains(os.Getenv("TEST_TYPE"), "both") {
-		shared.LogLevel("info", "TEST: 'NEW' Secrets Encryption method")
-		secretsEncryptOps("rotate-keys", product, cpIp, nodes)
-	}
+	secretsEncryptOps("prepare", product, cluster.ServerIPs[0], cpIp, nodes)
+	secretsEncryptOps("rotate", product, cluster.ServerIPs[0], cpIp, nodes)
+	secretsEncryptOps("reencrypt", product, cluster.ServerIPs[0], cpIp, nodes)
+	secretsEncryptOps("rotate-keys", product, cluster.ServerIPs[0], cpIp, nodes)
 }
 
-func secretsEncryptOps(action, product, cpIP string, nodes []shared.Node) {
+func secretsEncryptOps(action, product, primaryEtcdIp, cpIP string, nodes []shared.Node) {
 	shared.LogLevel("info", "TEST: Secrets-Encryption:  "+action)
 	_, errStatusB4 := shared.SecretEncryptOps("status", cpIP, product)
 	Expect(errStatusB4).NotTo(HaveOccurred(), "error getting secret-encryption status before action")
 
 	stdOutput, err := shared.SecretEncryptOps(action, cpIP, product)
 	Expect(err).NotTo(HaveOccurred(), "error: secret-encryption: "+action)
 	verifyActionStdOut(action, stdOutput)
-	if (action == "reencrypt") || (action == "rotate-keys") {
-		shared.LogLevel("DEBUG", "reencrypt op needs some time to complete - Sleep for 20 seconds before service restarts")
-		time.Sleep(20 * time.Second) // Wait for reencrypt action to complete before restarting services.
-	}
+
+	shared.LogLevel("debug", "secrets-encrypt ops need to complete - Sleep for 30 seconds before service restarts")
+	time.Sleep(30 * time.Second)
+
+	// Restart Primary Etcd Node First
+	restartServerAndWait(primaryEtcdIp, product)
```

Comment:
Can we add debug log here before and after restart like `Restart service -> Starting...` `Restart service -> Completed!`